angular.json

```diff
@@ -60,18 +60,47 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ],
-              "optimization": true,
-              "outputHashing": "all",
-              "sourceMap": false,
+              "optimization": false,
+              "outputHashing": "none",
+              "sourceMap": true,
               "namedChunks": false,
-              "extractLicenses": true,
+              "showCircularDependencies": false,
+              "aot": true,
+              "extractLicenses": false,
+              "statsJson": false,
+              "progress": false,
               "vendorChunk": false,
-              "buildOptimizer": true,
+              "buildOptimizer": false,
+              "subresourceIntegrity": false,
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "5mb",
-                  "maximumError": "8mb"
```

Comment:
not sure we should change these -> only the local build should be affected by the changes, not the drone build so the difference in maximum build size doesn't make sense imo